examples/README.md

```diff
@@ -9,7 +9,23 @@ You may want to read through the
 [Quick Start Guide](https://grpc.io/docs/quickstart/java.html)
 before trying out the examples.
 
-To build the examples,
+## Basic examples
+
+- [Hello world](src/main/java/io/grpc/examples/helloworld)
+
+- [Route guide](src/main/java/io/grpc/examples/routeguide)
+
+- [Metadata](src/main/java/io/grpc/examples/header)
+
+- [Error handling](src/main/java/io/grpc/examples/errorhandling)
+
+- [Compression](src/main/java/io/grpc/examples/experimental)
+
+- [Flow control](src/main/java/io/grpc/examples/manualflowcontrol)
+
+- [Json serialization](src/main/java/io/grpc/examples/advanced)
+
+#### To build the above examples,
```

Comment:
This is an abuse of a header. 1) choosing a header size instead of a header depth (why is there no `###`) and 2) having a header that is part of a sentence that is continued below.

I'd accept `### To build the examples`

Reply:
Done.